cassandra/query.py

```diff
@@ -996,7 +996,8 @@ def populate(self, max_wait=2.0, wait_for_complete=True, query_cl=None):
                 SimpleStatement(self._SELECT_SESSIONS_FORMAT, consistency_level=query_cl), (self.trace_id,), time_spent, max_wait)
 
             # PYTHON-730: There is race condition that the duration mutation is written before started_at the for fast queries
-            is_complete = session_results and session_results[0].duration is not None and session_results[0].started_at is not None
+            session_row = session_results.one() if session_results else None
+            is_complete = session_results and session_row.duration is not None and session_row.started_at is not None
```

Comment:
It seems slightly odd to look at session_results again here.  Should this perhaps be:

```python
is_complete = session_row is not None and session_row.duration is not None and session_row.started_at is not None
```
What do you think?

Reply:
Yes, absolutely, it makes more sense to check `session_row` here, thanks